core/drivers/crypto/se050/crypto.mk

```diff
@@ -38,6 +38,12 @@ ifeq ($(CFG_NXP_SE05X_SCP03_DRV),y)
 $(call force,CFG_SCP03_PTA,y,Mandated by CFG_NXP_SE05X_SCP03)
 endif
 
+# Allow a secure client to send APDU raw frames
+CFG_NXP_SE05X_APDU_DRV ?= y
+ifeq ($(CFG_NXP_SE05X_APDU_DRV),y)
+$(call force,CFG_APDU_PTA,y,Mandated by CFG_NXP_SE05X_APDU)
```

Comment:
I dont think that it is a better choice; I mean when we add support for another secure element, the commits will not be consistent. 

in one case,  this will have been added in "core: pta: APDU pseudo trusted application"
CFG_APDU_PTA ?= $(CFG_NXP_SE05X_APDU_DRV)
$(eval $(call cfg-depends-all,CFG_APDU_PTA,CFG_NXP_SE05X_APDU_DRV))

and in another case (hypotetical Xilinx device) this would hav e been added in "crypto: xxxx: APDU driver"

CFG_APDU_PTA ?= $(CFG_NXP_XILINX_XXX_APDU_DRV)
$(eval $(call cfg-depends-all,CFG_APDU_PTA,CFG_XILINX_XXX_APDU_DRV))

I think it is better as is. But I can change it if you want

Reply:
In the hypothetical case of another driver for another APDU device, wouldn't we end up with `$(eval $(call cfg-depends-one,CFG_APDU_PTA,CFG_APDU_DRV1 CFG_APDU_DRV2))`?
Anyways it doesn't matter much for now.